config/ember-try.js

```diff
@@ -1,74 +1,90 @@
 'use strict';
 
 const getChannelURL = require('ember-source-channel-url');
+const { embroiderSafe, embroiderOptimized } = require('@embroider/test-setup');
 
-module.exports = async function() {
+module.exports = async function () {
   return {
     useYarn: true,
     scenarios: [
       {
         name: 'ember-lts-3.4',
         npm: {
           devDependencies: {
-            'ember-source': '~3.4.0'
-          }
-        }
+            'ember-source': '~3.4.0',
+          },
+        },
       },
       {
         name: 'ember-lts-3.8',
         npm: {
           devDependencies: {
-            'ember-source': '~3.8.0'
-          }
-        }
+            'ember-source': '~3.8.0',
+          },
+        },
+      },
+      {
+        name: 'ember-lts-3.20',
+        npm: {
+          devDependencies: {
+            'ember-source': '~3.8.0',
+          },
+        },
       },
       {
         name: 'ember-release',
         npm: {
           devDependencies: {
-            'ember-source': await getChannelURL('release')
-          }
-        }
+            'ember-source': await getChannelURL('release'),
+          },
+        },
       },
       {
         name: 'ember-beta',
         npm: {
           devDependencies: {
-            'ember-source': await getChannelURL('beta')
-          }
-        }
+            'ember-source': await getChannelURL('beta'),
+          },
+        },
       },
       {
         name: 'ember-canary',
         npm: {
           devDependencies: {
-            'ember-source': await getChannelURL('canary')
-          }
-        }
+            'ember-source': await getChannelURL('canary'),
+          },
+        },
```

Comment:
These three scenarios, plus the 2 embroider scenarios, all need to include webpack and ember-auto-import v2 as devDependencies as well.

I suggest defining that at the top of this file as 
```js
const ember4AddtlDeps = {
  'ember-auto-import': '^2',
  'webpack': '^5',
};
```

and then update the configs here to be e.g.

```js
npm: {
  devDependencies: {
    ...ember4AddtlDeps,
    'ember-source': await getChannelURL(...),
  },
},
```